img/t/tiketti.fi.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="194.872" height="194.871" viewBox="0 0 146.154 146.153"><defs><clipPath id="a"><path d="M0 0h841.89v595.276H0Z"/></clipPath><clipPath id="b" clipPathUnits="userSpaceOnUse"><path d="M54.444 201.062H236.12v196.234H54.444z" style="fill:#000;stroke-width:.75"/></clipPath></defs><g clip-path="url(#a)" transform="translate(-71.65 -223.195)"><path fill="#dfa726" d="M169.009 369.348c26.948 0 48.795-21.846 48.795-48.795 0-26.948-21.847-48.794-48.795-48.794s-48.795 21.846-48.795 48.794c0 26.949 21.847 48.795 48.795 48.795"/><path fill="#dfa726" d="M71.65 320.553c0-26.87 21.722-48.667 48.564-48.792v97.584c-26.842-.125-48.563-21.92-48.563-48.792m48.563-48.794v-48.563c-26.765.125-48.437 21.797-48.56 48.563zm48.798-48.564c26.87 0 48.667 21.721 48.792 48.563H120.22c.125-26.842 21.92-48.563 48.792-48.563"/><path fill="#231f20" d="M579.45 325.683c.25-2.13.42-4.288.42-6.49 0-29.285-22.752-53.024-50.815-53.024s-50.814 23.739-50.814 53.024 22.442 53.025 52.866 53.025c22.196 0 40.047-13.083 47.896-31.79l-21.923-6.945c-4.207 11.16-13.66 18.19-25.973 18.19-15.306 0-26.664-10.02-29.05-25.99h53.648zm-24.49-17.445h-52.18c3.4-13.898 13.576-22.151 26.275-22.151 12.632 0 22.533 8.547 25.904 22.15m-260.654 28.494v-46.356h20.718v-21.13h-20.718v-30.564h-21.991v15.277h-.01c0 .036.01.069.01.102 0 8.464-6.826 15.111-15.276 15.166v.01h-4.783v21.139h18.46v50.255c0 18.666 11.487 28.731 29.948 28.731 8.616 0 14.565-.523 14.565-.523v-19.896s-4.924.302-8.206.302c-8.82 0-12.717-3.692-12.717-12.513m405.569 0v-46.356h20.718v-21.13h-20.718v-30.564h-21.99v15.277h-.01c0 .036.01.069.01.102 0 8.464-6.827 15.111-15.277 15.166v.01h-4.783v21.139h18.461v50.255c0 18.666 11.486 28.731 29.947 28.731 8.616 0 14.566-.523 14.566-.523v-19.896s-4.925.302-8.207.302c-8.819 0-12.717-3.692-12.717-12.513m-71.489 0v-46.356h20.718v-21.13h-20.718v-30.564h-21.991v15.277h-.01c0 .036.01.069.01.102 0 8.464-6.826 15.111-15.276 15.166v.01h-4.783v21.139h18.46v50.255c0 18.666 11.487 28.731 29.948 28.731 8.616 0 14.565-.523 14.565-.523v-19.896s-4.924.302-8.206.302c-8.82 0-12.717-3.692-12.717-12.513m-191.988-25.229 41.025-42.256h-31.59l-38.358 40.41v-70.973h-23.589v130.459h23.59v-27.898l12.307-12.922 29.332 40.82h29.128zm-99.918 57.64h23.589v-99.896h-23.589zm-3.693-130.357c0 8.615 6.974 15.385 15.385 15.385 8.615 0 15.589-6.77 15.589-15.385s-6.974-15.59-15.59-15.59c-8.41 0-15.384 6.975-15.384 15.59m409.262 130.357h23.589v-99.896h-23.589zm-3.693-130.357c0 8.615 6.974 15.385 15.385 15.385 8.615 0 15.59-6.77 15.59-15.385s-6.975-15.59-15.59-15.59c-8.41 0-15.385 6.975-15.385 15.59" clip-path="url(#b)"/></g></svg>
```

Comment:
There is still a bit of  superfluous stuff in there, so here's an optimized version:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 146.154 146.153"><path fill="#dfa726" d="M97.359 146.153c26.948 0 48.795-21.846 48.795-48.795 0-26.948-21.847-48.794-48.795-48.794S48.564 70.41 48.564 97.358c0 26.949 21.847 48.795 48.795 48.795M0 97.358c0-26.87 21.722-48.667 48.564-48.792v97.584C21.722 146.025.001 124.23.001 97.358m48.563-48.794V.001C21.799.126.127 21.798.004 48.564ZM97.362 0c26.87 0 48.667 21.721 48.792 48.563H48.57C48.695 21.721 70.49 0 97.362 0"/></svg>
```